spec/requests/api_competitions_spec.rb

```diff
@@ -549,10 +549,14 @@
 
         it "prevents from CSRF attacks" do
           headers["ACCESS_TOKEN"] = "INVALID"
```

Comment:
Aaaah. This test case exactly reveals what has been my gut all along:
We absolutely cannot merge this feature in its current state. You're basically giving anybody who tries to fake a CSRF token the opportunity to get issued a valid token and retry.

The core idea of this test is that "if you supply an invalid token, then you shall not pass". This is good and necessary, it's called "web security". Re-issuing a token when something invalid gets submitted implicitly breaks our whole security system and cannot go live.